### PR TITLE
BCDA-777: Swagger is updated to include Coverage endpoint

### PR DESCRIPTION
**Pull request by tbellj**
Fixes [BCDA-777](https://jira.cms.gov/browse/BCDA-777)

Problem:
New endpoint so our documentation needs to be updated.

Proposed changes:
API.md - Added documentation for our user guide.
bcda/api.go - Added coverage documentation for swagger

Security Implications:
None

Acceptance Validation:
Check the swagger documentation as well as our user guide.

Feedback Requested:
Any

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -589,3 +589,164 @@ The response will be the requested data as FHIR Patient resources in NDJSON form
 	}
 }
 ```
+
+### Beneficiary Coverage Data
+	The process of retrieving coverage data is very similar to all of the other exporting operations supported by this api.
```
Comment (embh) [inline API.md]:
```suggestion
	The process of retrieving coverage data is very similar to all of the other exporting operations supported by this API.
```

File: API.md [hunk 2]
```diff
@@ -589,3 +589,164 @@ The response will be the requested data as FHIR Patient resources in NDJSON form
...
+	-H 'Authorization: Bearer {token}' \
+	-H 'Accept: application/fhir+json' \
+	-H 'Prefer: respond-async'
+	```
+
+  ##### Response
+	If the request was successful, a `202 Accepted` response code will be returned and the response will include a `Content-Location` header. The value of this header indicates the location to check for job status and outcome. In the example header below, the number 43 in the URL represents the ID of the export job.
+
+  ###### Headers
+	* `Content-Location: https://sandbox.bcda.cms.gov/api/v1/jobs/43`
```
Comment (embh) [inline API.md]: Should we use a different job ID for this example? Even though they're meant to be standalone, the other two examples have different IDs (42, 43), so to be on the safe side maybe we want to use 44 or some other number for this one.

Comment (tbellj) [reply API.md]: good point, will do! and good catch

File: bcda-site-static/user_guide.md [hunk 3]
```diff
@@ -624,3 +624,164 @@ The response will be the requested data as FHIR Patient resources in NDJSON form
...
+	curl https://sandbox.bcda.cms.gov/data/44/DBBD1CE1-AE24-435C-807D-ED45953077D3.ndjson \
+	-H 'Authorization: Bearer {token}'
+	```
+
+  ##### Response
+	The response will be the requested data as FHIR Coverage resources in NDJSON format. An example of one such resource is shown below.
+
+  ```json
+  {
+  "fullUrl": "https:///v1/fhir/Coverage/part-a-19990000002901",
```
Comment (embh) [inline bcda-site-static/user_guide.md]: Is there supposed to be a domain in this URL?

File: bcda-site-static/user_guide.md [hunk 4]
```diff
@@ -624,3 +624,164 @@ The response will be the requested data as FHIR Patient resources in NDJSON form
 	}
 }
 ```
+
```
Comment (msnook) [inline bcda-site-static/user_guide.md]: The user guide changes looks like a duplication of the work that was done in BCDA-1077 ( #225).  Is there anything different in here then in that work?

Comment (knollfear) [reply bcda-site-static/user_guide.md]: I think the documentation in api.go is new.  That will add this to swagger

Comment (tbellj) [reply bcda-site-static/user_guide.md]: yep just saw that. Other than the changes done to bcda/api.go I think everything else is a duplication. Im a do a pull now that #225 is merged and it should resolve.

File: bcda-site-static/user_guide.md (hunk 3 above)
Comment (tbellj) [reply bcda-site-static/user_guide.md]: I have no idea

Comment (msnook) [reply bcda-site-static/user_guide.md]: Looks like it is the same for Patient and EOB on sandbox.bcda.cms.gov:
```
"fullUrl": "https:///v1/fhir/Patient/19990000002901",
```

File: bcda/api.go [hunk 5]
```diff
@@ -104,6 +104,24 @@ func bulkPatientRequest(w http.ResponseWriter, r *http.Request) {
...
+
+	Produces:
+	- application/fhir+json
+
+	Security:
+		api_key
+
+	Responses:
+		202:BulkRequestResponse
+		400:ErrorModel
```
[the remark this answers is not in the document]
Comment (tbellj) [reply bcda/api.go]: done

**Pull request merged**